config/locales/subscriber_authentication.yml

```diff
@@ -40,3 +40,10 @@ en:
         <p class="govuk-body">You’ve set up a GOV.UK account for <strong>%{address}</strong></p>
         <p class="govuk-body">You now need to use your email address and password to manage your notifications.</p>
         <p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="%{process_govuk_account_path}">Sign in to your account</a>.</p>
+    no_subscriber_account:
```

Comment:
Let's use `no_subscriber` instead of `no_subscriber_account` here (and elsewhere) - the term "subscriber account" isn't used anywhere else, we just talk about "subscribers".